Replace calls to IsPullRequestBuild

Replace calls to IsPullRequestBuild with proper check if build is for a pull request:

```
D:/a/1/s/tools/Cake.Issues.Recipe.0.2.1/Content/build.cake(132,55): error CS1061: 'IssuesData' does not contain a definition for 'IsPullRequestBuild' and no accessible extension method 'IsPullRequestBuild' accepting a first argument of type 'IssuesData' could be found (are you missing a using directive or an assembly reference?)
```

--- a/Cake.Issues.Recipe/Content/build.cake
+++ b/Cake.Issues.Recipe/Content/build.cake
@@ -129,7 +129,7 @@ IssuesBuildTasks.CreateSummaryIssuesReportTask = Task("Create-SummaryIssuesRepor
 IssuesBuildTasks.ReportIssuesToPullRequestTask = Task("Report-IssuesToPullRequest")
     .Description("Report issues to pull request.")
     .WithCriteria(() => IssuesParameters.PullRequestSystem.ShouldReportIssuesToPullRequest, "Reporting of issues to pull requests is disabled")
-    .WithCriteria<IssuesData>((context, data) => data.IsPullRequestBuild, "Not a pull request build")
+    .WithCriteria<IssuesData>((context, data) => data.BuildServer != null ? data.BuildServer.DetermineIfPullRequest(context) : false, "Not a pull request build")
     .IsDependentOn("Read-Issues")
     .Does<IssuesData>((data) =>
 {
@@ -145,7 +145,7 @@ IssuesBuildTasks.ReportIssuesToPullRequestTask = Task("Report-IssuesToPullReques
 IssuesBuildTasks.SetPullRequestIssuesStateTask = Task("Set-PullRequestIssuesState")
     .Description("Set pull request status.")
     .WithCriteria(() => IssuesParameters.PullRequestSystem.ShouldSetPullRequestStatus, "Setting of pull request status is disabled")
-    .WithCriteria<IssuesData>((context, data) => data.IsPullRequestBuild, "Not a pull request build")
+    .WithCriteria<IssuesData>((context, data) => data.BuildServer != null ? data.BuildServer.DetermineIfPullRequest(context) : false, "Not a pull request build")
     .IsDependentOn("Read-Issues")
     .Does<IssuesData>((data) =>
 {

